pkg/scheduler/objects/utilities_test.go

```diff
@@ -49,11 +49,20 @@ func createManagedQueue(parentSQ *Queue, name string, parent bool, maxRes map[st
 
 // create managed queue with props set
 func createManagedQueueWithProps(parentSQ *Queue, name string, parent bool, maxRes, props map[string]string) (*Queue, error) {
+	return createManagedQueuePropsMaxApps(parentSQ, name, parent, maxRes, props, uint64(0))
```

Comment:
Does `maxApps` have to be an `uint64`? I mean I understand that it's more logical, but now we have these weird casts here and in queue.go:1359. Simple `int` is the most portable data type.